src/ui/webeiddialog.cpp

```diff
@@ -109,7 +109,17 @@ void WebEidDialog::showAboutPage()
 {
     WebEidDialog* d = new WebEidDialog();
     d->setAttribute(Qt::WA_DeleteOnClose);
-    d->ui->okButton->hide();
+    d->ui->helpButton->hide();
+    d->ui->aboutAlert->hide();
+    if (qApp->applicationName() == QStringLiteral("web-eid-safari")) {
```

Comment:
Can this code be simplified by moving it into a method `setupAboutOK()` in the `Application` class? You would get rid of the custom event and define the About dialog OK slot `onAboutOK()` instead in the `Application` class and connect the signal to the slot in `setupAboutOK()`. Exposing `d` to `Application` gets messy though, so this needs a bit more thought and experimenting.

Reply:
This is opened from Controller